src/Backoffice/Task/App/Requests/UpsertTaskRequest.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Lightit\Backoffice\Task\App\Requests;
+
+use Illuminate\Foundation\Http\FormRequest;
+use Illuminate\Validation\Rule;
+use Lightit\Backoffice\Task\Domain\DataTransferObjects\TaskDto;
+use Lightit\Backoffice\Task\Domain\Enums\TaskStatus;
+
+class UpsertTaskRequest extends FormRequest
+{
+    public const TITLE = 'title';
+
+    public const DESCRIPTION = 'description';
+
+    public const STATUS = 'status';
+
+    public const EMPLOYEE_ID = 'employee_id';
+
+    public const TASK_ID = 'id';
+
+    /**
+     * @return array<string, mixed>
+     */
+    public function rules(): array
+    {
+        return [
+            self::TITLE => ['required', 'string', 'min:2', 'max:80'],
+            self::DESCRIPTION => ['required', 'string', 'min:2'],
+            self::STATUS => ['required', Rule::enum(TaskStatus::class)],
+            self::EMPLOYEE_ID => ['required', 'exists:employees,id'],
+            self::TASK_ID => ['sometimes', 'exists:tasks,id'],
```

Comment:
I suggest doing this instead

```suggestion
            self::EMPLOYEE_ID => ['required', Rule::exists(Employee::class, 'id')],
            self::TASK_ID => ['sometimes', Rule::exists(Task::class, 'id')],
```

The reason is that, every time we have a `Rule` method for validating something, we use it. Additionally, you can use the class directly so the method takes the table's name right from the model